examples/connectors-native-sdk-example-java-github-connector/README.md

```diff
@@ -0,0 +1,17 @@
+# Snowflake Native SDK for Connectors GitHub Example
+
+This example shows how to implement a simple connector application based on the [template](../../templates). This connector
+uses GitHub API to retrieve the data about issues from a repository. This example includes all the basic steps of the Native SDK for Connectors flow.
+
+Consult the step by step [quickstart](https://quickstarts.snowflake.com/guide/connectors_native_sdk_java_example) to learn more about the GitHub example.
+
+You can find the [template here](../../templates) and the implementation tutorial in the [documentation](link to tutorial)
```

Comment:
Maybe this sentence is redundant. We already have a link to the template (README) where there is information about the tutorial in the 3. line

Reply:
it's on purpose